src/components/Button.tsx

```diff
@@ -10,7 +10,7 @@ export default function Button({
     return (
         <Link
             href={link}
-            className="p-3 w-fit border-[3px] border-red-800 rounded-lg duration-75 ease-out hover:px-4 cursor-pointer"
+            className="p-3 w-fit ml-4 border-[3px] border-red-800 rounded-lg duration-75 ease-out hover:px-4 cursor-pointer"
```

Comment:
generally better to avoid styling directly in these top-level components bc this specific style isnt needed across all instances - a good solution could be to add another parameter that lets u set a dynamic margin, or just more styles in general